benchmark/src/datasets/ac_branin.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+import itertools
+
+import numpy as np
+import pandas as pd
+from sklearn import datasets
+from src.datasets.dataset import Dataset
+
+
+class ACBraninDataset(Dataset):
+    def __init__(self) -> None:
+        self._data = pd.read_csv("/home/benjamin/Dokumente/code/tmp/SMAC3/benchmark/src/datasets/ac_branin_instances_features.csv", skipinitialspace=True)
```

Comment:
you probably will not want to load this from your computer :)

Reply:
done